### DISCUSSION TITLE
for intro documentation

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#1893] [Q&A]
Can someone explain, or direct me to the right part of documentation, exactly what the Pkg and RdTbl do? They seem to be dependent on the active window but it's not entirely clear to me what precisely the various options do. 

pkg is the package and rdtbl is the readtale.
 
MAN WHO-LINE

**Comment by nbriggs** [comment]
It should be in lispusers/who-line.tedit - they are supposed to set the package/readtable for the current TTY process

**Comment by pamoroso** [comment]
Are there any tasks or fixes associated with this issue? Should it be converted to a discussion?

**Comment by pamoroso** [comment]
I will convert this issue to a discussion.
